test/model_validation_test.py

```diff
@@ -15,7 +15,10 @@
 
 import unittest
 
-import model.concourse as examinee
+from model import (
```

Comment:
as discussed, please adhere to our naming convention for tests. split the tests into separate modules, each named as the module under test with a suffix '_test'. e.g. `model/concourse.py` --> `test/model/concourse_test.py`

Reply:
done